Telie/Controllers/CreateReactionViewController.swift

```diff
@@ -187,7 +189,7 @@ class CreateReactionViewController: UIViewController {
         episodeTextLabel.snp.makeConstraints { make in
             make.top.equalTo(episodeLabel.snp.bottom).offset(6)
             make.leading.equalToSuperview().offset(leadingTrailingPadding)
-            make.trailing.equalTo(downButton.snp.leading).offset(leadingTrailingPadding)
+            make.trailing.equalTo(downButton.snp.leading).offset(-10)
```

Comment:
we love hardcode

Reply:
LOL i got lazy and didn't want to make a new variable